src/modules/aws_cloudfront/entity/distribution.ts

```diff
@@ -188,4 +189,20 @@ export class Distribution {
     nullable: true,
   })
   status?: string;
+
+  /**
+   * @public
+   * Associate a certificate from AWS Certificate Manager. The certificate must be in the US East (N. Virginia) Region (us-east-1).
```

Comment:
hmm, interesting, why is this? We might need to add a constraint or a trigger to check it.

Reply:
That's AWS' constraint. Seems like `CHECK` [isn't able](https://stackoverflow.com/a/27107470/2594912) to enforce a cross-table constraint. However, it's possible to do it using `TRIGGER`s. I'll do it as a follow-up.